### PR TITLE
Fix Multiple Overrides issue

### PR DESCRIPTION
**Pull request by jameszezhan**
Fixes multiple overrides issue. #34 
This update would return an array of overridden files instead of just one overridden file.

Also updated unittest to reflect the changes

### CODE REVIEW

File: Test/Unit/Model/RunnerTest.php [hunk 1]
```diff
@@ -58,33 +58,54 @@ public function testRun()
             'diff -r Magento-EE-2.3.1/vendor/magento/module-checkout/view/frontend/web/js/view/billing-address.js Magento-EE-2.3.3/vendor/magento/module-checkout/view/frontend/web/js/view/billing-address.js',
             'diff -r Magento-EE-2.3.1/vendor/magento/module-bundle/view/frontend/templates/js/components.phtml Magento-EE-2.3.3/vendor/magento/module-bundle/view/frontend/templates/js/components.phtml',
             'diff -r Magento-EE-2.3.1/vendor/magento/module-checkout/view/frontend/web/template/billing-address/details.html Magento-EE-2.3.3/vendor/magento/module-checkout/view/frontend/web/template/billing-address/details.html',
+            'diff -r Magento-EE-2.3.1/vendor/magento/module-customer/view/frontend/templates/form/forgotpasswords.phtml Magento-EE-2.3.3/vendor/magento/module-customer/view/frontend/templates/form/forgotpasswords.phtml',
```
Comment (mpchadwick) [inline Test/Unit/Model/RunnerTest.php:61]: Build is failing for me locally. I think this all instances should be `forgotpassword.phtml` here (not `forgorpasswords.phtml`)

File: Test/Unit/Model/RunnerTest.php [hunk 2]
```diff
@@ -93,7 +114,46 @@ public function testRun()
             'SomethingDigitalUpgradeHelper\Module\Controller\Adminhtml\Promo\Catalog\Save'
         );
 
+        // Preference
```
Comment (mpchadwick) [inline Test/Unit/Model/RunnerTest.php:117]: This isn't a preference. Can you add a comment to describe what we're actually testing here (override of same file in 2 themes).

File: Console/UpgradeHelperCommand.php [hunk 3]
```diff
@@ -36,7 +36,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $output->writeln('-------- ' . $type . ' --------');
             foreach ($items as $patched => $customized) {
                 $output->writeln('Patched: ' . $patched);
-                $output->writeln('Customized: ' . $customized);
+                if (is_array($customized)) {
+                    foreach ($customized as $customizedFiile) {
+                        $output->writeln('Customized: ' . $customizedFile);
```
Comment (mpchadwick) [inline Console/UpgradeHelperCommand.php:41]: This can't work you has `$customizedFiile` in the loop but then refer to it as `$customizedFile`

**Review by mpchadwick — CHANGES_REQUESTED** [review]

File: Console/UpgradeHelperCommand.php (hunk 3 above)
Comment (jameszezhan) [reply Console/UpgradeHelperCommand.php:41]: oops, missed 1/2 typo

File: Test/Unit/Model/RunnerTest.php (hunk 2 above)
Comment (jameszezhan) [reply Test/Unit/Model/RunnerTest.php]: This is updated with a new commit

File: Test/Unit/Model/RunnerTest.php (hunk 1 above)
Comment (jameszezhan) [reply Test/Unit/Model/RunnerTest.php]: Updated this as well

**Review by mpchadwick — APPROVED** [review]

**Pull request merged**